config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: test
+
+production:
```

Comment:
Are we in fact using Cable...?

Reply:
pretty sure no, it's just autogenerated

Reply:
yeah, we shouldn't be - just autogenerated

I can try removing it and hope everything works? :D